src/Microsoft.AspNet.Mvc.Core/ModelBinding/KeyValuePairModelBinder.cs

```diff
@@ -15,8 +15,13 @@ public sealed class KeyValuePairModelBinder<TKey, TValue> : IModelBinder
                                                       typeof(KeyValuePair<TKey, TValue>),
                                                       allowNullModel: true);
 
-            var keyResult = await TryBindStrongModel<TKey>(bindingContext, "Key");
-            var valueResult = await TryBindStrongModel<TValue>(bindingContext, "Value");
+            var validationNode = new ModelValidationNode(
+                bindingContext.ModelName,
+                bindingContext.ModelMetadata,
+                model: null);
```

Comment:
node itself is never used; instead create an empty `List<ModelValidationNode>` and pass _that_ to `TryBindStrongModel`